packages/social-previews/src/helpers.js

```diff
@@ -18,3 +18,9 @@ export const firstValid =
 
 export const stripHtmlTags = ( description ) =>
 	description ? description.replace( /(<([^>]+)>)/gi, '' ) : '';
+
+export const formatTweetDate = new Intl.DateTimeFormat( 'en-US', {
```

Comment:
Wouldn't you want to use `default` here or to ensure this doesn't render in English for other locales?